### ISSUE TITLE
Frequencies with sector splits

### ISSUE THREAD

**Issue by becas22** [pedro2555/messagemaker#76]
ATIS instructs to contact 125.55, even if you have a sector split (LPPC_N_CTR)
Frequencies: https://docs.google.com/spreadsheets/d/1xxHEd4dX4Yuwrhm48uebc-K-AVtkA7jFautui4jLw-A/edit#gid=0

**Comment by pedro2555** [comment]
This is most likelly a configuration issue.

## TLDR
Update the settings file [src/settings.py#L46](https://github.com/pedro2555/messagemaker/blob/master/src/settings.py) `clr_freq` and `dep_freq` options for all airports affected.

## Feature explanation and rational
There are two different classs of frequencies considered:
 * Clearance
 * Departure

Configurable per airport, in the [settings](https://github.com/pedro2555/messagemaker/blob/master/src/settings.py) file, using the entries:
 * `clr_freq`
 * `dep_freq`

respectively.

The behaviour is to show the frequency for each classification when that frequency does not match the information on the charts (_exception being Lisboa, where even 119.1 is shown_). This is the behaviour of the real ATIS, as far as I recall.

The implementation, however, is very straight forward. The first frequency from a list that is online is shown (works the same for _clearance_ and _departure_ frequencies). It is a matter of having the correct frequencies in the correct order.

As an example the current list of _clearance frquencies for Liaboa can be found in [src/settings.py#L46](https://github.com/pedro2555/messagemaker/blob/4a2b17a29edb52cd8557855925ce2701aaf636cc/src/settings.py#L46)
```python
'clr_freq': (
    ('118.950', '[FOR DEP CLEARANCE CONTACT DEL 118.950]'),
    ('121.750', '[FOR DEP CLEARANCE CONTACT GND 121.750]'),
    ('118.100', '[FOR DEP CLEARANCE CONTACT TWR 118.100]'),
    ('119.100', '[ON THE GROUND CONTACT APP 119.1]'),
    ('125.550', '[ON THE GROUND CONTACT 125.55]'),
)
```

_Note. All messages that mention the position (DEL, GND, TWR, etc) are so because they are in the real ATIS. Messages that do not mention the position are not in real ATIS (_125.55 is never the departure frequency, but can be on VATSIM_)._

**Comment by pedro2555** [comment]
closed in f34a72314f983ef858d1c7dc71fe62b7d696928b
